examples/src/main/java/org/wfanet/examples/streaming/HelloStreamingClient.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2021 The Cross-Media Measurement Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wfanet.examples.streaming;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.stub.StreamObserver;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+import org.wfanet.examples.streaming.HelloStreamingGrpc.HelloStreamingStub;
+
+class HelloStreamingClient {
+  private static final Logger logger = Logger.getLogger(HelloStreamingClient.class.getName());
+
+  public static void main(String[] args) throws Exception {
+    String target = "localhost:50051";
+    ManagedChannel channel = ManagedChannelBuilder.forTarget(target).usePlaintext().build();
+    HelloStreamingStub asyncStub = HelloStreamingGrpc.newStub(channel);
+    try {
```

Comment:
Suggestion: pull the contents of this `try` block out into a helper function to reduce nesting